homeassistant/components/isy994/__init__.py

```diff
@@ -55,6 +56,14 @@
 )
 
 
+async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
+    """Set up the ISY 994 integration."""
+
+    async_setup_services(hass)
```

Comment:
Note: we now recommend to keep the services registered, and you should consider dropping `async_unload_services` in `isy994`

Reply:
Separate PR or update this one ?